tests/epics/adaravis/test_aravis.py

```diff
@@ -123,6 +123,7 @@ async def test_can_collect(
         "dataset": "/entry/data/data",
         "swmr": False,
         "multiplier": 1,
+        "chunk_size": 0,
```

Comment:
Can we `set_mock_value` on `test_adaravis.hdf.num_frames_chunk` so we get a realistic number please?